python/spec/dependabot/python/update_checker/requirements_updater_spec.rb

```diff
@@ -14,7 +14,7 @@
     )
   end
 
-  let(:update_strategy) { :bump_versions }
+  let(:update_strategy) { "bump_versions" }
```

Comment:
On L391 of this file you need to change

```diff
- %i(bump_versions bump_versions_if_necessary).each do |update_strategy|
+ %w(bump_versions bump_versions_if_necessary).each do |update_strategy|
```

and that should fix the python spec failures in CI.

Reply:
Thanks for the fix!